src/main.tsx

```diff
@@ -1,92 +1,66 @@
-import { Guan4Tou2MemberPage } from './member/2024/Guan4Tou2.tsx'
-import { CHWMemberPage } from './member/2023/CHW.tsx'
-
-const router = createHashRouter(
-  createRoutesFromElements(
-      <Route element={<RootLayout></RootLayout>}>
-        <Route path='/' element={<Home></Home>}></Route>
-        <Route path='/Member' element={<Member></Member>}></Route>
-        <Route path='/Member/2023/Uriah' element={<UriahMemberPage />}></Route>
-        <Route path='/Member/2023/Salt' element={<SaltMemberPage />}></Route>
-        <Route path='/Member/2022/Ryan' element={<RyanMemberPage />}></Route>
-        <Route path='/Member/2024/Adb2' element={<Adb2MemberPage />}></Route>
-        <Route path='/Member/2024/WIFI' element={<WIFIMemberPage />}></Route>
-        <Route path='/Member/2023/Eason' element={<EasonMemberPage />}></Route>
-        <Route path='/Member/2024/ypp' element={<YPPMemberPage />}></Route>
-        <Route path='/Member/2024/Young' element={<YoungMemberPage />}></Route>
-        <Route path='/Member/2024/Jason' element={<JasonMemberPage />}></Route>
-        <Route path='/Member/2023/Bobo' element={<BoboMemberPage />}></Route>
-        <Route path='/Member/2024/PaulWang' element={<PaulWangMemberPage />}></Route>
-        <Route path='/Member/2024/Joe' element={<JoeMemberPage />}></Route>
-        <Route path='/Member/2024/Guan4Tou2' element={<Guan4Tou2MemberPage />}></Route>
-        <Route path='/Member/2023/CHW' element={<CHWMemberPage />}></Route>
-        <Route path='/Alumni' element={<Alumni></Alumni>}></Route>
-        <Route path='/Partner' element={<Partner></Partner>}></Route>
-        <Route path='/Profile' element={<Profile></Profile>}></Route>
-        <Route path='/Publication' element={<Publication></Publication>}></Route>
-        <Route path='/Experience' element={<Experience></Experience>}></Route>
-        <Route path='/Record' element={<Record></Record>}></Route>
-        <Route path='/News' element={<News></News>}></Route>
-        <Route path='/Activities' element={<Activities></Activities>}></Route>
-        <Route path='/Rules' element={<Rule></Rule>}></Route>
-      </Route>
-  )
-)
-
-ReactDOM.createRoot(document.getElementById('root')!).render(
-  <React.StrictMode>
-    <ConfigProvider theme={{
-      components: {
-        Menu: {
-          darkItemColor: "white",
-          darkItemSelectedBg: "black",
-          darkItemBg: "black",
-          darkSubMenuItemBg: "black",
-          subMenuItemBg: "black",
-          darkItemSelectedColor: "#cf2e2e"
-        },
-        Table: {
-          colorBgContainer: "black",
-          colorTextDescription: "white",
-          colorText: "white",
-          colorBorder: "white",
-          headerColor: "white",
-          borderColor: "white",
-          headerBorderRadius: 0
-        }
-      }
-    }}>
-      <RouterProvider router={router}></RouterProvider>
-    </ConfigProvider>
-  </React.StrictMode>,
-)
```

Comment:
Why it disappear? It may happen when you fix the conflict and remove it accdently. Please, fix it.